### DISCUSSION TITLE
Open browser's bookmark in new window

### DISCUSSION THREAD

**Discussion by vbccbz** [Flow-Launcher/Flow.Launcher#2550] [Q&A] [answered]
It would be nice to have an option.

**Comment by jjw24** [comment, marked as answer]
You can set this in settings under general tab, something called 'Default Browser' or similar, which will give you the option to open as new window browser or tab.

**Reply by vbccbz** [reply]
Yes, thx! And to complete create one-level hierarchy through all windows there is extension New Tab, New Window for Chrome.
